bin/ch/Helpers.cpp

```diff
@@ -556,7 +557,15 @@ void GetBinaryPathWithFileNameA(char *path, const size_t buffer_size, const char
     char dir[_MAX_DIR];
 
     char modulename[_MAX_PATH];
-    PlatformAgnostic::SystemInfo::GetBinaryLocation(modulename, _MAX_PATH);
+    charcount_t pathLength;
+
+    bool result = ChPlatformAgnostic::Module::GetBinaryLocation(modulename, _countof(modulename) - 1, &pathLength);
```

Comment:
>_countof(modulename) - 1 [](start = 76, length = 24)

nit: The `- 1` here is different to elsewhere and a bit confusing on what GetBinaryLocation behaves. Remove it if not needed?